src/main/java/com/fasterxml/jackson/databind/ser/PropertyWriter.java

```diff
@@ -21,7 +21,7 @@ public abstract class PropertyWriter
     extends ConcreteBeanPropertyBase // since 2.7
     implements java.io.Serializable
 {
-    private static final long serialVersionUID = 1L;
+    protected static final long serialVersionUID = 1L;
```

Comment:
this looks wrong to me - I think this will have a Java serialization impact

Reply:
I reverted this change